src/main/java/seedu/elisa/logic/commands/SnoozeCommand.java

```diff
@@ -0,0 +1,117 @@
+package seedu.elisa.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.elisa.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.elisa.logic.parser.CliSyntax.PREFIX_SNOOZE;
+
+import java.time.LocalDateTime;
+import java.util.NoSuchElementException;
+
+import seedu.elisa.commons.core.Messages;
+import seedu.elisa.commons.core.index.Index;
+import seedu.elisa.commons.core.item.Item;
+import seedu.elisa.logic.commands.exceptions.CommandException;
+import seedu.elisa.model.ItemModel;
+import seedu.elisa.model.item.VisualizeList;
+
+/**
+ * Edits the details of an existing item in the item list.
+ */
+public class SnoozeCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "snooze";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Snoozes the reminder "
+            + "by the index number used in the reminder list "
+            + "or the most recent occurred reminder if no index is provided.\n"
+            + "Parameters: [INDEX] (must be a positive integer) "
+            + "[" + PREFIX_SNOOZE + "SNOOZE TIME]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_SNOOZE + "1.min.later";
+
+    public static final String MESSAGE_SNOOZED_REMINDER_SUCCESS = "Snoozed Reminder: %1$s,"
+            + " because someone is real lazy...";
+
+    private static final String SHOW_REMINDER_VIEW = "R";
+
+    private final boolean hasIndex;
+    private final Index index;
+    private final LocalDateTime newReminderOccurrence;
+
+    private Item oldItem;
+    private Item snoozedItem;
+
+    /**
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+     * @param hasIndex boolean that indicates if an index was specified.
+     * @param index of the item to edit.
+     * @param newReminderOccurrence LocalDateTime of new occurrence of reminder.
+=======
+     * @param index of the item to edit.
+     * @param newReminderOccurrence LocalDateTime of new occurrence of reminder
+>>>>>>> Halfway adding SnoozeCommand
+=======
+     * @param hasIndex boolean that indicates if an index was specified.
+     * @param index of the item to edit.
+     * @param newReminderOccurrence LocalDateTime of new occurrence of reminder.
+>>>>>>> Add snooze command
+=======
+     * @param hasIndex boolean that indicates if an index was specified.
+     * @param index of the item to edit.
+     * @param newReminderOccurrence LocalDateTime of new occurrence of reminder.
+>>>>>>> 77beef877620bb25d68066b2550360272c1e9c07
+     */
```

Comment:
This looks to be a problem with the merge conflict. Perhaps we can edit it in the next iteration.